index/segment/mem/terms_dict_bench_test.go

```diff
@@ -82,29 +70,11 @@ func BenchmarkTermsDict(b *testing.B) {
 }
 
 func benchmarkInsertSimpleTermsDict(docs []doc.Document, b *testing.B) {
+	dict := newSimpleTermsDict(NewOptions().SetInitialCapacity(10))
 	b.ReportAllocs()
+	b.ResetTimer()
 
 	for n := 0; n < b.N; n++ {
-		b.StopTimer()
-		dict := newSimpleTermsDict(NewOptions())
```

Comment:
If we move the initialization out of the benchmark won't every iteration after the first be a no-op?

Reply:
Fair. I thought you were testing something else (allocs for repeated inserts). Will undo change.